flowetl/tests/unit/conftest.py

```diff
@@ -20,3 +42,78 @@ def unload_airflow():
         for module in list(sys.modules.keys()):
             if "airflow" in module or "flowetl" in module:
                 del sys.modules[module]
+
+
+# In hindsight, these could have been a factory or partial. Ho hum.
+@pytest.fixture()
+def dummy_db_conn(postgresql_db, monkeypatch):
+    postgresql_db.install_extension("file_fdw")
```

Comment:
This is so the tests can be run against not-flowdb?

Reply:
Yep - testing the operators individually, not the flowdb integration.